### ISSUE TITLE
Issues with Skills and Effects sections - no break into multiple lines

### ISSUE THREAD

**Issue by apoapostolov** [Larkinabout/fvtt-token-action-hud-pf2e#35]
Skills and Effects sections seem not to break into multiple lines in 1.3.1

Tried to give the section a 100-200 pixels width to see if it would be affected, but it was ignored. 

![image](https://user-images.githubusercontent.com/4083812/230741632-6864f9f9-afcc-4202-a7bc-62ec9d7a55c6.png)

**Comment by Larkinabout** [comment]
Yeah, I need to fix something in TAH Core. It seems to work some of the time.
